DotaStats/AppDelegate.swift

```diff
@@ -2,14 +2,30 @@ import UIKit
 
 @main
 class AppDelegate: UIResponder, UIApplicationDelegate {
-
+    
     var window: UIWindow?
-
+    var tabBarController: UITabBarController = {
+        let tabBar = UITabBarController()
```

Comment:
Может лучше назвать так:
```suggestion
        let tabBarController = UITabBarController()
```
А то ниже, как мне кажется, странно: tabBar.tabBar